packages/@okta/vuepress-site/docs/guides/oie-embedded-common-org-setup/main/index.md

```diff
@@ -7,7 +7,22 @@ title: Get set up
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" /><br>
 
-## Overview
+<StackSelector class="cleaner-selector"/>
+
+This guide covers how to create and set up your Okta org before you can run the provided sample app or integrate the SDK or Widget into your own app.
+
+---
+**Learning outcomes**
```

Comment:
Again, needs blank lines inserted, to avoid rendering weirdness. I'll not comment on this every time, but you should do it for each one.

Reply:
Fixed